addons/website_event/static/src/js/website_event.js

```diff
@@ -44,8 +44,8 @@ var EventRegistrationForm = Widget.extend({
         $('#registration_form select').each(function () {
             post[$(this).attr('name')] = $(this).val();
         });
-        var tickets_ordered = Object.values(post).map((value) => parseInt(value))?.length > 0;
-        if (!tickets_ordered) {
+        var no_tickets_ordered = Object.values(post).map((value) => parseInt(value)).every(value => value === 0);
+        if (no_tickets_ordered) {
```

Comment:
Seems good, looks like `_.some` was not converted properly.

Looks like the warning insert doesn't actually work either... and has been broken at least since 14.0 woops. Not sure if we want to fix or remove, we can make that an improvement in another task after talking with lna though ^^